File: Code 3 Callouts/Code 3 Callouts/Models/Peds/Suspect.cs
using Rage;
using static Stealth.Plugins.Code3Callouts.Common;

namespace Stealth.Plugins.Code3Callouts.Models.Peds
{

	internal class Suspect : PedBase
	{

		internal Suspect(string pName, Rage.Vector3 position, bool confirmedSuspect) : base(pName, PedType.Suspect, position)
        {
			if (confirmedSuspect == false) {
				this.Type = PedType.Unknown;
			}
		}

		internal Suspect(string pName, Rage.Model model, Rage.Vector3 position, float heading, bool confirmedSuspect) : base(pName, PedType.Suspect, model, position, heading)
        {
			if (confirmedSuspect == false) {
				this.Type = PedType.Unknown;
			}
		}

		protected internal Suspect(string pName, Rage.PoolHandle handle, bool confirmedSuspect) : base(pName, PedType.Suspect, handle)
        {
			if (confirmedSuspect == false) {
				this.Type = PedType.Unknown;
			}
		}

	}

}

File: Code 3 Callouts/Code 3 Callouts/Models/Peds/Suspect.vb
Imports Rage

Namespace Models.Peds

    Public Class Suspect
        Inherits PedBase

        Public Sub New(ByVal pName As String, ByVal position As Rage.Vector3, ByVal confirmedSuspect As Boolean)
            MyBase.New(pName, PedType.Suspect, position)

            If confirmedSuspect = False Then
                Me.Type = PedType.Unknown
            End If
        End Sub

        Public Sub New(ByVal pName As String, ByVal model As Rage.Model, ByVal position As Rage.Vector3, ByVal heading As Single, ByVal confirmedSuspect As Boolean)
            MyBase.New(pName, PedType.Suspect, model, position, heading)

            If confirmedSuspect = False Then
                Me.Type = PedType.Unknown
            End If
        End Sub

        Protected Friend Sub New(ByVal pName As String, ByVal handle As Rage.PoolHandle, ByVal confirmedSuspect As Boolean)
            MyBase.New(pName, PedType.Suspect, handle)

            If confirmedSuspect = False Then
                Me.Type = PedType.Unknown
            End If
        End Sub

    End Class

End Namespace